client/css/style.css

```diff
@@ -1840,6 +1846,10 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	flex: 0 0 auto;
 	border: 1px solid transparent;
 	transition: border-color 0.2s;
+
+	/* TODO: Get rid of this once networks can be edited
+					 See https://github.com/thelounge/thelounge/pull/2229 */
+	display: none;
```

Comment:
Can we exclude this from this PR please.

Reply:
Yes of course, I should have written WIP or REMOVE and not TODO, I meant to leave it untouched, but while I didn't want to have it get in the way or appear on the screenshot while I was iterating on this. I am not planning to work on its style since it's meant to go, but yes I will remove it from this PR :)